pkg/npm/npm.go

```diff
@@ -146,6 +146,7 @@ func (exec *Execute) RunScriptsInAllPackages(runScripts []string, runOptions []s
 
 	if len(packagesList) > 0 {
 		packageJSONFiles = packagesList
+		log.Entry().Info("Using provided package-list: " + strings.Join(packageJSONFiles, ", "))
```

Comment:
in go fmt at least you can add it like so:
```
fmt.Println("first string", "second string") // strings are stitched together with spaces
```
Looks to me like it is the same with [this Logrus Info function](https://pkg.go.dev/github.com/sirupsen/logrus#Info), i.e.:

```suggestion
		log.Entry().Info("Using provided package-list:", strings.Join(packageJSONFiles, ", "))
```

Reply:
```suggestion
		log.Entry().Infof("Using provided package-list: %s", strings.Join(packageJSONFiles, ", "))
```
would be even better